styles/atom-import-cost.less

```diff
@@ -7,5 +7,5 @@
   opacity: 0.5;
   margin-left: 10px;
   position: relative;
-  top: -(@component-line-height - (@component-padding/2));
+  top: -@component-line-height;
```

Comment:
yeah, in custom font, it breaks a bit bad :P (but its fine) people hardly change it ;)